packages/app/src/components/settings/NotificationSettings.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Button, Grid, Typography } from '@material-ui/core';
+import {
+  discoveryApiRef,
+  errorApiRef,
+  fetchApiRef,
+  identityApiRef,
+  useApi,
+} from '@backstage/core-plugin-api';
+
+import { InfoCard } from '@backstage/core-components';
+import React from 'react';
+import { UserNotificationSettingsCard } from '@backstage/plugin-notifications';
+
+const TEST_NOTIFICATION = {
+  recipients: { type: 'entity', entityRef: 'user:default/guest' },
+  payload: {
+    title: 'Test Notification',
+    severity: 'normal',
+    topic: 'Setting Test',
+  },
+};
+
+export const NotificationSettings = () => {
+  const fetchApi = useApi(fetchApiRef);
+  const discovery = useApi(discoveryApiRef);
+  const identityApi = useApi(identityApiRef);
+  const errorApi = useApi(errorApiRef);
+
+  const handleNotifyClick = async () => {
+    const notificationUrl = await discovery.getBaseUrl('notifications');
+    const token = await identityApi.getCredentials();
+    const response = await fetchApi.fetch(notificationUrl, {
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+      method: 'POST',
+      body: JSON.stringify(TEST_NOTIFICATION),
+    });
+    if (!response.ok) {
```

Comment:
Not matter what I always seem to hit a 401 🤔

Reply:
Ends up this only works with service to service and not use flows. Need to refactor my approach.